testing/live/DeeperDeepDungeonDex/manifest.toml

```diff
@@ -0,0 +1,5 @@
+[plugin]
```

Comment:
This was due to feature creep, initially I just wanted the smaller cropped images as thumbnails, then we found that it would be nice to show the full pictures on click.

Once I have more time I plan remove the thumbnails and calculate modified UV coordinates to display thumbnails.

Reply:
Addressed in v1.0.0.1